python/social_bot/gazebo_agent.py

```diff
@@ -54,6 +58,13 @@ def __init__(self,
                 if None it will be set the same as agent_type
             config (dict): the configuarations for the agent
                 see `agent_cfg.jason` for details
+            use_pid_control (bool): whether use pid controller for agent.
+                Note: not all agents have pid control setup.  Check agent_cfg.json.
+            use_simple_full_states (bool): Use the simplest full states like
+                agent's distance and direction to goal
+            view_angle_limit (float): the radian angle to limit the agent's observation
```

Comment:
better to use degree than radian. It's easier to specify

Reply:
Done